tests/Fixer/Import/GroupImportFixerTest.php

```diff
@@ -0,0 +1,240 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\Import;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Volodymyr Kupriienko <vldmr.kuprienko@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\Import\GroupImportFixer
+ */
+final class GroupImportFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string $expected
+     *
+     * @dataProvider provideDoNotFixCases
```

Comment:
No need for data provider as there is one case and there won't be more.

Reply:
Fixed.